index.bs

```diff
@@ -701,6 +701,18 @@ the UA MUST run the following steps:
   5. Add |device| to |allowedDevice|@{{[[devices]]}}.
   6. Return |allowedDevice|.
 
+To <dfn data-lt="remove device from storage">remove an allowed <a>USB
+device</a></dfn> |device|, given a {{USBPermissionStorage}} |storage|,
+the UA MUST run the following steps:
+
+  1. Search for an element |allowedDevice| in
+     <code>|storage|.{{USBPermissionStorage/allowedDevices}}</code> where
+     |device| is in |allowedDevice|@{{[[devices]]}}, if no such element exists,
+     abort these steps.
+  2. Remove |device| from |allowedDevice|@{{[[devices]]}}.
```

Comment:
Is this step necessary since `allowedDevice` is being removed completely?

Reply:
Good catch!